decoder/jws.go

```diff
@@ -51,7 +52,13 @@ func (d *jwsDecoder) Decode(ctx context.Context, rawJws string) (*Token, error)
 			if strVal, ok := value.(string); ok {
 				token.Claims[destKey] = strVal
 			} else {
-				return nil, UnexpectedClaimTypeError{key, value}
+				strJson, err := json.Marshal(value)
```

Comment:
the linter is complaining that this should be `strJSON`, would you mind changing it?

Reply:
Fixed the linter warning (sorry it looks I forgot to enable `golint`)

Reply:
Thank you!